torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <torch/nn/options/activation.h>
+#include <torch/types.h>
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor linear(const Tensor& input, const Tensor& weight,
+                     const Tensor& bias) {
```

Comment:
`bias` can be `None` in Python, and the equivalent default is `Tensor()` (aka. undefined tensor) in C++.